src/test/java/hw5/pages/UserTablePage.java

```diff
@@ -0,0 +1,95 @@
+package hw5.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class UserTablePage {
+
+    protected WebDriver driver;
+
+    @FindBy(tagName = "select")
+    private List<WebElement> dropDownList;
+    @FindBy(css = "#user-table a")
+    private List<WebElement> userNameList;
+    @FindBy(css = "#user-table span")
+    private List<WebElement> descriptionList;
+    @FindBy(css = "input[type='checkbox']")
+    private List<WebElement> checkboxList;
+    @FindBy(xpath = "//tr[2]//option")
+    private List<WebElement> dropDownItemsRoman;
+    @FindBy(id = "ivan")
+    private WebElement checkboxForIvan;
+    @FindBy(css = "ul.panel-body-list.logs li")
+    private List<WebElement> logsList;
+
+    public UserTablePage(WebDriver driver) {
+        PageFactory.initElements(driver, this);
+        this.driver = driver;
+    }
+
+    public String getPageTitle() {
+        return driver.getTitle();
+    }
```

Comment:
May be it will be better extract this method to the base class?

Reply:
Done. Moved to the BasePage class for 3 pages.